lib/checks/navigation/region-evaluate.js

```diff
@@ -86,6 +91,14 @@ function findRegionlessElms(virtualNode, options) {
   }
 }
 
+function isShallowlyHidden(virtualNode) {
+  // The element itself is not visible to screen readers, but its descendants might be
+  return (
+    getRole(virtualNode, { noPresentational: true }) === null &&
```

Comment:
Talked to @dbjorge about this. We think this is the way to do it:

```suggestion
    ['none', 'presentation'].includes(getRole(virtualNode)) &&
```

This avoids us skipping elements like `<canvas aria-label="foo">`. That's not something I think we should skip. Same with focusable stuff. This change limits it to elements that explicitly have role="none", or have role=none implicitly (which only happens for img).